### PR TITLE
set screen resolution for emulator

### PR DESCRIPTION
**Pull request by greenEkatherine**

### CODE REVIEW

**Comment by greenEkatherine** [comment]
Eventually, I've set up supported screen resolution without "UI not responding" error

File: azure-pipelines.yml [hunk 1]
```diff
@@ -33,12 +33,12 @@ jobs:
         steps:
         - bash: |
             echo "y" | $ANDROID_HOME/tools/bin/sdkmanager --install 'system-images;android-${{ value }};google_apis;x86'
-            echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-${{ value }};google_apis;x86' --force
+            echo "no" | $ANDROID_HOME/tools/bin/avdmanager create avd -n xamarin_android_emulator -k 'system-images;android-${{ value }};google_apis;x86' -d 9 --force
```
Comment (omen273) [inline azure-pipelines.yml:36]: Please, add comment why -d 9

File: azure-pipelines.yml [hunk 2]
```diff
@@ -33,12 +33,12 @@ jobs:
...
 
             $ANDROID_HOME/emulator/emulator -list-avds
 
             echo "Starting emulator"
-            nohup $ANDROID_HOME/emulator/emulator -avd xamarin_android_emulator -no-snapshot > /dev/null 2>&1 &
+            nohup $ANDROID_HOME/emulator/emulator -avd xamarin_android_emulator -skin 1080x1920 -no-snapshot > /dev/null 2>&1 &
```
Comment (omen273) [inline azure-pipelines.yml:41]: We need different resolution for different devices. From 720x1280 to 1440x3040. You can see all possible resolutions in AVD manager. It is about a new task.

**Review by omen273 — APPROVED** [review]

File: azure-pipelines.yml (hunk 2 above)
Comment (greenEkatherine) [reply azure-pipelines.yml:41]: opened an issue https://github.com/omen273/crosswordApp/issues/64

File: azure-pipelines.yml (hunk 1 above)
Comment (greenEkatherine) [reply azure-pipelines.yml:33]: done

**Pull request merged**